app.json

```diff
@@ -41,6 +41,9 @@
     "PULSE_TASKS_QUEUE_NAME": {
       "generator": "secret"
     },
+    "PULSE_QUEUE_NAME": {
```

Comment:
With Heroku review apps (`app.json` configures them) we can switch over more easily since we start from a clean slate every time.

Please remove `PULSE_RESULSETS_QUEUE_NAME` and `PULSE_TASKS_QUEUE_NAME`.

Reply:
Under formation, remove "pulse_listerer_pushes" and "pulse_listerner_tasks" and add the new one.

Reply:
Done. Can you please tell me what exactly do we intend to achieve from this?